package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "KeePassXC-Browser",
-  "version": "1.7.10.1",
+  "version": "1.8.0",
```

Comment:
We have a dependabot alert for ansi-regex to be updated to >= 5.0.1

Reply:
I know. It's a dependency of a dependency. Gotta solve it somehow.